### PR TITLE
CR for #20530 - Add DDE calibration

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: afce981d83a3f4bf117781ae4081c989c0add015 available at: https://gist.github.com/df11dee467b469e27081<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4786.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4786.dmg

**Comment by cozza13** [comment]
this is great. Can you update with master. @ctrlaltdavid

**Comment by hifi-gustavo** [comment]
Build console output for commit: cac6ae114814f0e7e50075aa3d4d4b43dd63787f available at: https://gist.github.com/df11dee467b469e27081<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4786.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4786.dmg

**Comment by cozza13** [comment]
This looks good.

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -3200,12 +3201,12 @@ void Application::displaySide(Camera& theCamera, bool selfAvatarOnly, RenderArgs
...
-                PerformanceWarning warn(Menu::getInstance()->isOptionChecked(MenuOption::PipelineWarnings),
-                    "Application::displaySide() ... atmosphere...");
-                _environment.renderAtmospheres(theCamera);
+            // draw the sky dome
+            if (!selfAvatarOnly && Menu::getInstance()->isOptionChecked(MenuOption::Atmosphere)) {
+                PerformanceTimer perfTimer("atmosphere");
+                PerformanceWarning warn(Menu::getInstance()->isOptionChecked(MenuOption::PipelineWarnings),
+                    "Application::displaySide() ... atmosphere...");
+                _environment.renderAtmospheres(theCamera);
             }
```
Comment (AndrewMeadows) [inline interface/src/Application.cpp]: Thank you for removing these bad line endings.

**Pull request merged**